lib/json_matchers/payload.rb

```diff
@@ -15,6 +15,8 @@ def to_s
     def extract_json_string(payload)
       if payload.respond_to?(:body)
         payload.body
+      elsif payload.respond_to?(:to_h)
```

Comment:
@seanpdoyle I didn't notice you changed the logic here from my #58 PR. This fails if the payload is an `Array`, which as a `#to_h` method on it. 

With a `payload.is_a?(Hash)` check from my PR things pass fine.

With the change here in 0.7.1:

```ruby
    Failure/Error: expect(payload).to match_response_schema('homes')

     TypeError:
       wrong element type Hash at 0 (expected array)
```

Example of what data is coming through as "payload" in this case (mock data):

```
[{"updated"=>"2017-12-08", "uuid"=>"0987623-1f4c-4a9d-8462-f873gd7c766sb"}, {"updated"=>"2017-08-02", "uuid"=>"abc12345-1f4c-4a9d-8462-18664247652c"}]
```

Reply:
Could you pull down https://github.com/thoughtbot/json_matchers/pull/61 and try it out?